chef/cookbooks/ohai/recipes/default.rb

```diff
@@ -112,6 +112,9 @@
 # drop relatively big attributes that we know we won't use
 node.automatic_attrs["kernel"].delete("modules")
 
+# drop the collected Etc.passwd entries, these can be huge
+node.automatic_attrs["etc"].delete("passwd")
```

Comment:
why not disable the passwd plugin directly on the client.rb file?

`Ohai::Config[:disabled_plugins] << "passwd"`

Reply:
hmm, thats even nicer, but it also removes the groups. will check